Pipfile.lock

```diff
@@ -52,11 +52,11 @@
         },
         "django": {
             "hashes": [
-                "sha256:275bec66fd2588dd517ada59b8bfb23d4a9abc5a362349139ddda3c7ff6f5ade",
-                "sha256:939652e9d34d7d53d74d5d8ef82a19e5f8bb2de75618f7e5360691b6e9667963"
+                "sha256:6cdb98d464e5ffc9cbe3d3ca5ca74d61e69ce1784d5e15b85ceb19dee939f650",
+                "sha256:b74420065c5a3f7bcb07c0811b1308239c05989e29963ff39f7b5e9c1cacc195"
             ],
             "index": "pypi",
-            "version": "==2.1.7"
+            "version": "==2.2rc1"
```

Comment:
😱 

Whyyyyyyy is `--pre` globally scoped :(((((

Reply:
I didn't anticipate the Django 2.2 release "fixing" this (at least in short-term / isolation!)